### DISCUSSION TITLE
Any plans for "inputObjectType" support?

### DISCUSSION THREAD

**Discussion by nate-fur** [graphql-nexus/nexus-prisma#177] [General]
I am able to successfully generate model definitions using nexus-prisma, so that they can be forwarded to Nexus's _objectType_ function. In the attached example I have a User model defined in my schema.prisma file, and with nexus-prisma, I am correctly generating types needed in my graphQL layer for the User model.

<img width="488" alt="Screen Shot 2021-12-23 at 9 51 17 AM" src="https://user-images.githubusercontent.com/37522274/147256679-172a58e1-ce76-4c6d-8776-c52a3cc1d759.png">

<img width="717" alt="Screen Shot 2021-12-23 at 9 37 53 AM" src="https://user-images.githubusercontent.com/37522274/147256706-ce274576-4cd9-4d8d-b950-c84aed4778c6.png">

It would be useful to have nexus-prisma support Nexus's _inputObjectType_ API so that the same UserWhereInput typesafety that exists in Prisma Client (on my server) can exist for my graphQL client (frontend app). 

<img width="572" alt="Screen Shot 2021-12-23 at 9 44 09 AM" src="https://user-images.githubusercontent.com/37522274/147256955-d7e69d2a-8813-4419-9133-e3d1ea11d1c0.png">

The following screenshot shows code that isn't supported (ie. UserWhereInput is not exported in nexus-prisma), but would be my wish for additional nexus-prisma functionality:
<img width="803" alt="Screen Shot 2021-12-23 at 9 45 46 AM" src="https://user-images.githubusercontent.com/37522274/147256959-88fdffbc-e6d0-4e92-bc6a-0244f7e14691.png">

Are there any plans to support this use-case? Happy to answer any clarifying questions!

**Comment by miguelbogota** [comment]
This feature is very needed not only for `Where` but also `WhereUnique`, `Order`, `Filter`, `Update` and `Create`.

I know some conversation is happening [here](https://github.com/prisma/nexus-prisma/issues/163) but I haven't seen any result into it.

I hope they do it quickly.
